### PR TITLE
add unit test to dubbo-registry/dubbo-registry-api and dubbo-registry/dubbo-registry-default

### PR DESCRIPTION
**Pull request by lixiaojiee**
What is the purpose of the change

add unit test to dubbo-registry/dubbo-registry-api and dubbo-registry/dubbo-registry-default
#1690 
## Brief changelog

dubbo-registry/dubbo-registry-default/src/test/java/org/apache/dubbo/registry/dubbo/DubboRegistryTest.java

## Verifying this change

ut pass

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/incubator-dubbo/tree/master/dubbo-test).
- [ ] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: dubbo-registry/dubbo-registry-api/src/test/java/org/apache/dubbo/registry/retry/FailedRetryTaskTest.java [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.BDDMockito.mock;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class FailedRetryTaskTest {
```
Comment (carryxyh) [inline dubbo-registry/dubbo-registry-api/src/test/java/org/apache/dubbo/registry/retry/FailedRetryTaskTest.java]: Looks like this class is a supplement to FailbackRegistryTest some scenarios? Why not put this code in FailbackRegistryTest?

Comment (lixiaojiee) [reply dubbo-registry/dubbo-registry-api/src/test/java/org/apache/dubbo/registry/retry/FailedRetryTaskTest.java]: > Looks like this class is a supplement to FailbackRegistryTest some scenarios? Why not put this code in FailbackRegistryTest?

I checked the code again. The functions of this class and some of the functions of FailbackRegistryTest InvokerTest are repetitive. My intention was to test only failed retries, and i will delete this code.

**Comment by lovepoem** [comment]
Please merge origin/master to make ci success

**Comment by lixiaojiee** [comment]
> Please merge origin/master to make ci success

It has been processed.

**Comment by lovepoem** [comment]
LGTM

**Pull request merged**